hybrid/compartment_examples/inter_comp_call/secure-update_ddc/shared-update_ddc.S

```diff
@@ -13,3 +13,6 @@ executive_switch:
     cvtp     clr, lr
     b        switch_compartment
     ret      clr
+
+#define INTER_COMP_LEAK_DDC 1
```

Comment:
If we do keep this, I think a comment is in order, as to where it's used.

Reply:
As of 2edd256406a0480c26bb8051a33c7ba53fdc755a, this no longer exists (however, if we decide to go with the original approach, I can go back and document it)